### DISCUSSION TITLE
Loading FontAwesome AND another font (but still have icons)

### DISCUSSION THREAD

**Discussion by CoolFox29281819** [ocornut/imgui#6841] [New Users Build/Link/Run issues ONLY!]
Hey guys,

So it seems to be one or another - I cannot have fontAwesome ions AND a custom font. This is problably just me so please tell me how I can fix this!

Custom font + icon
![image](https://github.com/ocornut/imgui/assets/144345472/1e34cd1c-8345-4e2e-b5a3-8fb19638c6e4)

Default font  + icon
![image](https://github.com/ocornut/imgui/assets/144345472/f757b19d-eff6-4955-b4c9-7eee42368d5e)

Thanks again
(if it isnt obvious im new to imgui)

**Comment by sakiodre** [comment]
Have you tried to follow [the guide](https://github.com/ocornut/imgui/blob/master/docs/FONTS.md#using-icon-fonts)? Just replace `AddFontDefault()` with `AddFontFromFileTTF()` of your custom font.

**Reply by CoolFox29281819** [reply]
Thanks,

sorry I was using font pointers instead of just adding them lol
